qual_id/pattern.py

```diff
@@ -87,6 +88,7 @@ class Pattern:
         "pokemon": Pokemon(),
         "profession": Profession(),
         "programminglanguage": ProgrammingLanguage(),
+        "randomid": RandomId(),
```

Comment:
```suggestion
        "id": RandomId(),
```
I'd be happy with just 'id' as the url parameter, but feel free to disagree 🙂

Reply:
I have made this change. I kept the class name the same as I didn't want to step on anything system like.... maybe not an issue